### ISSUE TITLE
Bug: startTime doesn't set in Vue JS implementation

### ISSUE THREAD

**Issue by sts-ryan-holton** [Wikiki/bulma-calendar#225]
I can't seem to set `startTime`? When I try to specify it and open my dialog, it's still displayed as: `00:00`, my code is as follows, this is a Vue JS implementation:

```js
/**
 * Init custom datetime picker from Bulma
 */
initDateTimePicker: function () {
  var now = new Date()
  console.log(moment(now).format('HH:mm'))
  const calendar = bulmaCalendar.attach(this.$refs.calendarTrigger, {
    type: 'datetime',
    dateFormat: 'YYYY-MM-DD',
    timeFormat: 'HH:mm',
    startDate: moment(now).format('YYYY-MM-DD'),
    startTime: moment(now).format('HH:mm'),
    showHeader: false,
    showButtons: false,
    showFooter: false,
    displayMode: 'dialog'
  })[0]
  calendar.on('select',date => {
    console.log(date.data.value())
  });
}
```

**Comment by tommed** [comment]
FYI it works if you set it to a `typeof Date` object instead of a string, but the setter startDate afterwards still doesn't work (another ticket it already open) :-/ not sure if this component is actually usable tbh

**Comment by sts-ryan-holton** [comment]
That's a shame really isn't it

**Comment by michael-hack** [comment]
Should be fixed in 6.1.2, if not please open a new issue.
